src/Fixer/Basic/BracesFixer.php

```diff
@@ -169,9 +169,17 @@ protected function createConfigurationDefinition()
             ->getOption()
         ;
 
+        $positionAfterControlStructures = new FixerOptionBuilder('position_after_control_structures', 'whether the opening brace should be placed on "next" or "same" line after control structures not covered by position_after_functions_and_oop_constructs.');
```

Comment:
Yes, IMO option names are explicit enough about that. But having description explicit as well won't hurt after all, so leave the sentence as is if you prefer :)